velox/functions/prestosql/StringFunctions.h

```diff
@@ -267,4 +267,60 @@ struct StrLPosFunction : public StrPosFunctionBase<T, true> {};
 template <typename T>
 struct StrRPosFunction : public StrPosFunctionBase<T, false> {};
 
+template <typename T>
+struct LevenshteinDistanceFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<int64_t>& result,
+      const arg_type<Varchar>& left,
+      const arg_type<Varchar>& right) {
+    std::vector<int32_t> leftCodePoints = stringImpl::stringToCodePoint(left);
```

Comment:
Would it be possible to add a fast path for all-ASCII inputs?

Reply:
> Would it be possible to add a fast path for all-ASCII inputs?

Can you provide some details about fast path for ASCII inputs you are referring to?

Reply:
See https://facebookincubator.github.io/velox/develop/scalar-functions.html#all-ascii-fast-path

Reply:
> See https://facebookincubator.github.io/velox/develop/scalar-functions.html#all-ascii-fast-path

That's COOL! will update ascii with fast path!

Reply:
> Would it be possible to add a fast path for all-ASCII inputs?

Actually after some learning, I found it all-ASCII inputs not really making it "faster" here, because: `stringImpl::stringToCodePoint` is the function transforming string to int. And the underlying function converting char to int is `utf8proc_codepoint`, which is copied from duckdb.cpp and directly handle char as utf8 and I didn't find optimization for ASCII. 

So my understanding is that `LevenshteinDistanceFunction` is not able to be optimized for all-ASCII scenario, unless we create an underlying optimization util function like `ASCIIproc_codepoint`. Let me know if it make sense, looking forward to your feedbacks!